helpers/browser/driver.js

```diff
@@ -134,6 +136,25 @@ class ChromeProtocol {
     });
   }
 
+  /**
+   * @param {!string} selector
```

Comment:
style nit for future reference: you don't actually need a `!` for lowercase `string` since it's a primitive type and so not nullable by default, but it doesn't do anything bad